packages/core/core/src/types.ts

```diff
@@ -196,8 +196,8 @@ export interface UseCheckout
 }
 
 export interface UseReview<REVIEW, REVIEWS_SEARCH_PARAMS, REVIEW_ADD_PARAMS> {
-  search: (params?: REVIEWS_SEARCH_PARAMS) => Promise<void>;
-  addReview: (params: REVIEW_ADD_PARAMS) => Promise<void>;
+  search: (params?: REVIEWS_SEARCH_PARAMS, customQuery?: CustomQuery) => Promise<void>;
```

Comment:
This will not work. You have to create a type overload. Check example from `useProduct` or `useCart`. Arguments hint should show params and `customQuery` as optional - not all systems will handle graphQL queries. Also, why params here are optional? You have to pass some data (`id`) to get/serach review right?